libr/lang/p/qjs.c

```diff
@@ -435,6 +435,11 @@ static bool lang_quickjs_file(RLangSession *s, const char *file) {
 }
 
 static void *init(RLangSession *ls) {
+	if (ls == NULL) {
+		// XXX should return bool instead
+		return (void*)1;
+	}
+	r_return_val_if_fail (ls, NULL);
```

Comment:
Isn't this redundant? And conflicting with the if above?

Reply:
yeah leftover!